src/core/include/cesium/omniverse/FabricMaterialDefinition.h

```diff
@@ -36,7 +44,8 @@ class FabricMaterialDefinition {
     bool _hasVertexColors;
     bool _hasBaseColorTexture;
     std::vector<FeatureIdType> _featureIdTypes;
-    uint64_t _imageryLayerCount;
+    std::vector<OverlayType> _imageryOverlayTypes;
```

Comment:
I'm not sure if `_imageryOverlayTypes` belongs here. Only `_imageryOverlayRenderPipes` seems relevant to `FabricMaterial`.

Reply:
Similarly, I don't think `overlayTypes` is needed in `ImageryLayersInfo` since it seems to be used just to populate this field.

Reply:
Yes. This is much better. Having _imageryOverlayTypes is just cruft from when we hadn't decoupled how layers are rendered. It also required a change to the equality operator for FabricMaterialDefinition, which was incorrectly checking layer types instead of the rendering method.